packages/webpack-config/webpack.base.js

```diff
@@ -93,6 +93,10 @@ module.exports = (basePath) => {
           test: /\.css$/i,
           use: [MiniCssExtractPlugin.loader, "css-loader"],
         },
+        {
+          test: /\.(png|svg|jpg|jpeg|gif)$/i,
```

Comment:
These filenames are stable hashes, right?  The names aren't going to change unless we change the images themeselves?
![image](https://user-images.githubusercontent.com/8978655/226931590-613f769e-3859-48b8-bcb8-080c73c932c1.png)

Reply:
okay, so it is the same as the bundle filename. Meaning, it is a hash of the file itself so it should be stable. That being said, we can add the original filename if we want, like how we are with our chunks and bundles.

- https://webpack.js.org/configuration/output/#outputassetmodulefilename
- https://webpack.js.org/configuration/output/#outputfilename

Reply:
Cool, I double confirmed that the hash remains the same by:

1. `rm -rf dist`
2. `npm run build:fresh`
3. Check hash for an image on the page
4. Repeat steps 1-3

![Screenshot 2023-03-22 103305](https://user-images.githubusercontent.com/4127998/226938592-1c1bf969-2544-4d9a-a865-c85060fe352e.png)
![Screenshot 2023-03-22 103556](https://user-images.githubusercontent.com/4127998/226938597-c5013439-703f-41ac-bb80-39c4394af619.png)


- `background-image: url("/11dd19b67fa6015463ef.jpg");`
- `background-image: url("/11dd19b67fa6015463ef.jpg");`